### BIG differences, this is diff of wrong files.
### CLUS "Variable +, ++, +++"
### and CLHS topic selection, I don't find ti
### +++ CLUS
### --- CLHS

--- saved-pages-dpans3/cl:variables:repl-plus.txt	2017-04-27 21:58:45.000000000 +0100
+++ saved-pages-clus/cl:variables:repl-plus.txt	2017-04-27 21:58:45.000000000 +0100
@@ -1,4 +1,140 @@
-Please select which reference to + you intended
+Variable +, ++,
-FunctionVariable
\ No newline at end of file
+Value Type
+an object.
+Initial Value
+implementation-dependent.
+Description
+The variables +, ++, and +++ are maintained by the Lisp read-eval-print loop to save forms that were recently evaluated.
+The value of + is the last form that was evaluated, the value of ++ is the previous value of +, and +++ is the previous value of ++.
+Examples
+(+ 0 1)
+→
+1
+(- 4 2)
+→
+2
+(/ 9 3)
+→
+3
+(list + ++ +++)
+→
+((/ 9 3) (- 4 2) (+ 0 1))
+(defparameter *a* 1)
+→
+*A*
+(defparameter *b* 2)
+→
+*B*
+(defparameter *c* 3)
+→
+*B*
+(defparameter *d* (list *a* *b* *c*))
+→
+*D*
+(setf *a* 4 *b* 5 *c* 6 *d* (list a b c))
+→
+(4 5 6)
+(list *a* *b* *c*)
+→
+(4 5 6)
+(eval +++)
+→
+(1 2 3)
+#.`(,@++ *d*)
+→
+(1 2 3 (1 2 3))
+Affected By
+Lisp read-eval-print loop.
+See Also
+Variable -, Variable *, Variable/, {\secref\TopLevelLoop}
+Notes
+None.
\ No newline at end of file
